src/environment.h

```diff
@@ -9,11 +9,33 @@
 
 namespace librealsense
 {
+    // The extrinsics graph is a graph of profiles.
```

Comment:
Use the following wording in the opening.
The class stores and managed the inter-stream extrinsic information in a form of dis/connected graph.
The nodes of the graph represent the streaming profiles that the different sensors provide. The edge between each pair of nodes designates and stores the `extrinsic` info - the spatial location and orientation of one sensor in the coordinate system of the other.
In case of multiple physical devices, the graph is automatically populated with a set of disconnected clusters (per camera) that allows for inter-stream but not inter-camera transformation.

Reply:
Done